From be6b47462c33ecd7caff258137bc314b0257b588 Mon Sep 17 00:00:00 2001
From: Frederik Schwan <frederik@tty42.de>
Date: Wed, 4 Jun 2025 17:22:32 +0200
Subject: [PATCH] remove "DistroVersion" from fu test

Some distros don't report any DistroVersion here due to their rolling
release concept.
---
 src/fu-self-test.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/fu-self-test.c b/src/fu-self-test.c
index 96c5c8500..6cd6be8fc 100644
--- a/src/fu-self-test.c
+++ b/src/fu-self-test.c
@@ -7047,7 +7047,6 @@ fu_engine_report_metadata_func(gconstpointer user_data)
 	    "CompileVersion(org.freedesktop.fwupd)",
 	    "CpuArchitecture",
 	    "DistroId",
-	    "DistroVersion",
 	    "FwupdSupported",
 	    "RuntimeVersion(org.freedesktop.fwupd)",
 	    "SELinux",
-- 
2.49.0

